### PR TITLE
Upgrade from aspectj 1.8.9 to 1.8.10

### PR DESCRIPTION
**Pull request by cory-p-oncota**

### CODE REVIEW

**Comment by cory-p-oncota** [comment]
Updating to report that I have signed the CLA. I'm not too sure why the build is failing here, a local compile/test/publishLocal is working fine. I currently cannot seem to access the build details (probably company internet policies), but if you have any feedback I'd be glad to assist.

**Comment by sslavic** [comment]
AspectJ 1.8.10 compilation now requires JDK 8 (see [AspectJ 1.8.10 release notes](https://eclipse.org/aspectj/doc/released/README-1810.html)). Build fails since it performs AspectJ compilation and runs on default JDK on Travis CI, which is at the moment Oracle JDK 7. It might be enough to update https://github.com/sbt/sbt-aspectj/blob/master/.travis.yml gets to require Oracle JDK 8 (see https://docs.travis-ci.com/user/languages/java/#Testing-Against-Multiple-JDKs how to configure this). Can you please do it in same PR?

**Comment by cory-p-oncota** [comment]
I've tried about every permutation I could think of from https://docs.travis-ci.com/user/languages/java/#Testing-Against-Multiple-JDKs with little luck. Hopefully can get travis-ci unblocked from my network soon to see the output.

**Comment by sslavic** [comment]
Just
```
jdk:
   - oraclejdk8
```
should be enough.

Now the problem is that for JDK 8, Scala 2.11.x+ is needed.
Might be enough to specify latest 2.11.x Scala version in Travis build as in https://docs.travis-ci.com/user/languages/scala/#Choosing-Scala-versions-to-test-against

Might be needed and probably would make sense to drop building across sbt versions, drop 0.12 since it's obsolete, build only for 0.13.

**Comment by cory-p-oncota** [comment]
Got it all in sync after getting rid of all of the cross-builder dependencies. Will send in one more commit to clean up `.travis.yml`.

**Comment by sslavic** [comment]
Looks good to me. @pvlugter please have a look.

File: .travis.yml [hunk 1]
```diff
@@ -1,2 +1,6 @@
 language: scala
-script: sbt "^ publish-local" "^ scripted"
+script: sbt "publish-local"
```
Comment (pvlugter) [inline .travis.yml]: This no longer runs the scripted tests in the travis build. We still need the scripted tests.

File: build.sbt [hunk 2]
```diff
@@ -5,17 +5,11 @@ organization := "com.typesafe.sbt"
...
 publishMavenStyle := false
 
 bintrayOrganization := Some("sbt")
 bintrayRepository := "sbt-plugin-releases"
 bintrayPackage := name.value
 bintrayReleaseOnPublish := false
-
-crossBuildingSettings
-CrossBuilding.crossSbtVersions := Seq("0.12", "0.13")
-CrossBuilding.scriptedSettings
```
Comment (pvlugter) [inline build.sbt]: This needs to switch from the `sbt-cross-building` scriptedSettings to the regular scripted plugin and settings.

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,2 +1 @@
 addSbtPlugin("me.lessis"        % "bintray-sbt"        % "0.3.0")
-addSbtPlugin("net.virtual-void" % "sbt-cross-building" % "0.8.1")
```
Comment (pvlugter) [inline project/plugins.sbt]: Switch to:

```
libraryDependencies += "org.scala-sbt" % "scripted-plugin" % sbtVersion.value
```

File: build.sbt (hunk 2 above)
Comment (pvlugter) [inline build.sbt]: Switch to `scriptedSettings` from regular scripted plugin here and keep the scriptedLaunchOpts.

**Comment by pvlugter** [comment]
I've added a commit to this PR, to enable scripted tests again for sbt 0.13. Have also bumped the tested versions to Scala 2.12 and Akka 2.4.

The plugin could be updated to latest sbt 0.13 (didn't bump because of deprecations) with latest sbt syntax, sbt auto plugin, and so on.

**Comment by ivantopo** [comment]
Hello there! May I ask if there are any plans to release a version that includes this PR soon?

**Pull request merged**